### ISSUE TITLE
Issue With Data & Media Directory

### ISSUE THREAD

**Issue by DroWnThePoor** [ampache/ampache-docker#98]
Hello. I was hesitant to post this issue here, but in the Ampache Google Group I was referred to do so.
I've created a Docker Ampache via the docker compose method.
I've been able to go through the installation process via web so I'm assuming my permissions work correctly.

I'm running Docker inside of a VM that is running on the machine where my music is stored.
I have the music directory mounted via NFS into the VM, and I am able to see my music directories in /data/media via SSH.
In this image it says ./data/media is mounted at /media for music:
![image](https://user-images.githubusercontent.com/16339020/227671833-868d77cc-1bcd-4b30-9154-b4755dea921a.png)

I'm assuming that means the /data/media directory is mounted as a volume inside of the container at /media. Is this correct?
However, when I try to create a catalog the only path that succeeds is /media  ; however it doesn't find any music.
All of the sub-directories are where the music is stored.
Any of my music sub-folders that are inside of this directory result in  
![image](https://user-images.githubusercontent.com/16339020/227674458-342b1e24-a125-483f-8dbb-9863c18a59e3.png)

I've set the permissions to be the same as /data/config because it seems that it is able to read the contents of this directory.
But I'm unsure if I'm not understanding something Docker-specific or if it is NFS causing an issue or what.

**Comment by kuzi-moto** [comment]
I have a feeling you mounted your NFS share to `/data/media` but the docker-compose file is using `./data/media` which is a relative path referencing the directory your docket-compose file exists, such as `/home/user/ampache/data//media`. If this is the case simply remove the period from the configuration file.

**Comment by DroWnThePoor** [comment]
Thank you for replying.  I ended up ditching the NFS route for a shared directory via my hypervisor.
However I'm aware of relative vs. absolute paths.
But since reading this I decided to recheck how I had done it anyways, and I had mounted the share correctly.
In my VM(where Docker runs the Ampache container) it created /data in my home folder, and /media as a subdir there.
Via SSH I could navigate to this location and use ls to see the music directories there.
On my current working configuration I simply changed the compose to /mnt/hgfs/ampache.
And in the web interface setting the path to /media scans and catalogs all of the sub-folders as expected.
It could be that I may have needed to set something differently on the server side exportfs file. 

I'm guesing I should close this with a comment as I'm not able to troubleshoot further.
